virttest/qemu_monitor.py

```diff
@@ -1188,25 +1192,36 @@ def set_migrate_capability(self, state, capability):
 
         :param state: Bool value of capability.
         :param capability: capability which need to set.
+        :raise MonitorNotSupportedMigCapError: if the capability is unknown
         """
         cmd = "migrate_set_capability"
         self.verify_supported_cmd(cmd)
         value = "off"
         if state:
             value = "on"
         cmd += " %s %s" % (capability, value)
-        return self.cmd(cmd)
+        result = self.cmd(cmd)
+        if result != "":
+            raise MonitorNotSupportedMigCapError("Failed to set capability"
+                                                 "%s: %s" %
+                                                 (capability, result))
```

Comment:
Here you change the return form of the function and it maybe raise MonitorNotSupportedMigCapError to caller.
And set_migrate_capability() has been referenced by other functions which doesn't handle the exception.
So i advise you don't raise Exception and check the result in case.